src/base/utils.js

```diff
@@ -235,10 +235,34 @@ export function removeArrayItem(arr, item) {
   }
 }
 
+// HTML5 video element recycle storage object.
+// If playback recycleVideo flag is set, it store
+// one video element for each player instance created.
+const videoStore = {}
+
+export class VideoStore {
```

Comment:
For exemple, if move the logic in `HTML5Video` constructor, i may add it just after super() call and then store video element if none available or override current one by recycled ? (in this case the parent _ensureElement() method has created a video element for nothing, but maybe it is an acceptable micro overheat ?)